sklearn/neighbors/tests/test_lof.py

```diff
@@ -118,3 +123,14 @@ def test_n_neighbors_attribute():
                          "n_neighbors will be set to (n_samples - 1)",
                          clf.fit, X)
     assert_equal(clf.n_neighbors_, X.shape[0] - 1)
+
+
+def test__score_samples():
```

Comment:
we test `_score_samples` not `score_samples` so I'm not sure of the convention...